### PR TITLE
Modify title and Abstract

### PR DESCRIPTION
**Pull request by DarrenTa**
The proposed changes provide for emphasis of the fact that ChainLocks strengthens the consensus mechanism.  

The introduction of the term "post Nakamoto consensus" will help me in the literature.  People can use the term ChainLocks when they want to talk about the technology.  I can use post Nakamoto Consensus when referring to how the game theory is different.

### CODE REVIEW

File: dip-0008.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ of blocks and finding near-instant consensus on the longest valid/accepted
 chain. ChainLocks leverages LLMQ Signing Requests/Sessions to accomplish
 this.
 
+[The original proposal by Satoshi Nakamoto](https://www.bitcoin.com/bitcoin.pdf) 
+provided a novel solution to the Byzantine general's problem.  In some publications, 
+this new consensuses mechanism is called a Nakamoto Consensus.  ChainLocks provide for
```
Comment (PastaPastaPasta) [inline dip-0008.md]:
```suggestion
this new consensuses mechanism is called Nakamoto Consensus.  ChainLocks provide for 
```

Comment (PastaPastaPasta) [inline dip-0008.md]:
```suggestion
this new consensuses mechanism is called a Nakamoto Consensus.  ChainLocks provides for 
```
Although "ChainLocks" ends with an 's', it's normally been treated as a singular noun

**Comment by DarrenTa** [comment]
So further conversation is found here:

https://www.reddit.com/r/dashpay/comments/chplaa/the_consensus_mechanism_of_dash_has_changed_its/

The term Synergy Consensus was suggested.  I do like this term.  I'm expecting to respond to a reporter this week, and would like to have a term I can use that will be consistent through the literature.

File: dip-0008.md [hunk 2]
```diff
@@ -33,6 +33,12 @@ of blocks and finding near-instant consensus on the longest valid/accepted
 chain. ChainLocks leverages LLMQ Signing Requests/Sessions to accomplish
 this.
 
+[The original proposal by Satoshi Nakamoto](https://www.bitcoin.com/bitcoin.pdf) 
+provided a novel solution to the Byzantine general's problem.  In some publications, 
+this new consensuses mechanism is called a Nakamoto Consensus.  ChainLocks provide for 
+an advancement of this consensus mechanism which mitigates a 51% attack.  We call
```
Comment (PastaPastaPasta) [inline dip-0008.md]:
```suggestion
an advancement of this consensus mechanism which mitigates 51% attacks.  We call 
```
above sounds better imo

File: dip-0008.md [hunk 3]
```diff
@@ -33,6 +33,12 @@ of blocks and finding near-instant consensus on the longest valid/accepted
 chain. ChainLocks leverages LLMQ Signing Requests/Sessions to accomplish
 this.
 
+[The original proposal by Satoshi Nakamoto](https://www.bitcoin.com/bitcoin.pdf) 
+provided a novel solution to the Byzantine general's problem.  In some publications, 
+this new consensuses mechanism is called a Nakamoto Consensus.  ChainLocks provide for 
+an advancement of this consensus mechanism which mitigates a 51% attack.  We call 
+the updated consensus mechanism that ChainLocks provide a post Nakamoto consensus.
```
Comment (PastaPastaPasta) [inline dip-0008.md]:
```suggestion
the updated consensus mechanism that ChainLocks provides a post Nakamoto consensus.  
```

Comment (DarrenTa) [reply dip-0008.md]: Thank you!

File: dip-0008.md [hunk 4]
```diff
@@ -33,6 +33,12 @@ of blocks and finding near-instant consensus on the longest valid/accepted
 chain. ChainLocks leverages LLMQ Signing Requests/Sessions to accomplish
 this.
 
+[The original proposal by Satoshi Nakamoto](https://www.bitcoin.com/bitcoin.pdf) 
+provided a novel solution to the Byzantine general's problem.  In some publications,
```
Comment (UdjinM6) [inline dip-0008.md]:
```suggestion
provided a novel solution to the Byzantine general's problem. In some publications, 
```

File: dip-0008.md [hunk 5]
```diff
@@ -33,6 +33,12 @@ of blocks and finding near-instant consensus on the longest valid/accepted
 chain. ChainLocks leverages LLMQ Signing Requests/Sessions to accomplish
 this.
 
+[The original proposal by Satoshi Nakamoto](https://www.bitcoin.com/bitcoin.pdf) 
+provided a novel solution to the Byzantine general's problem.  In some publications, 
+this new consensuses mechanism is called a Nakamoto consensus.  ChainLocks provides for
```
Comment (UdjinM6) [inline dip-0008.md]:
```suggestion
this new consensuses mechanism is called a Nakamoto consensus. ChainLocks provides 
```

File: dip-0008.md [hunk 6]
```diff
@@ -33,6 +33,12 @@ of blocks and finding near-instant consensus on the longest valid/accepted
 chain. ChainLocks leverages LLMQ Signing Requests/Sessions to accomplish
 this.
 
+[The original proposal by Satoshi Nakamoto](https://www.bitcoin.com/bitcoin.pdf) 
+provided a novel solution to the Byzantine general's problem.  In some publications, 
+this new consensuses mechanism is called a Nakamoto consensus.  ChainLocks provides for 
+an advancement of this consensus mechanism which mitigates 51% attacks.  We call
```
Comment (UdjinM6) [inline dip-0008.md]:
```suggestion
an advancement of this consensus mechanism which mitigates 51% attacks. We call 
```

**Comment by DarrenTa** [comment]
I believe I have accepted changes from UdjinM6

**Comment by DarrenTa** [comment]
Having something to call it will be helpful.  It is not that important to me what it is called.

**Comment by nmarley** [comment]
Personally I’m not sure it makes sense to add new terms to the DIP for the sake of interviews or for academic reasons. I would think a simple, high-level description of the ChainLocks consensus mechanism is possible without creating our own term for it and adding it to this DIP.

**Comment by DarrenTa** [comment]
Thanks for the feedback @nmarley 

I'll get to writing that up.

**Comment by thephez** [comment]
@DarrenTa Thanks for thinking through ways to effectively communicate this technical content to a broader audience :+1: 

For what it's worth, I think it is okay to use some clarifying terminology when describing ChainLocks. Perhaps additional terms / descriptions will become common naturally in time (ala "Nakamoto Consensus"), but I do agree with @nmarley that it is probably unnecessary to incorporate such terms in a DIP. These documents are targeted mainly at the technical audience that will grasp the consensus implications.

**Pull request closed without merging**